plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -275,13 +275,15 @@ const createNav = async ({ args, createNodeId, nodeModel, locales }) => {
     },
   });
 
-  const allNavYamlNodes = nodeModel.getAllNodes({ type: 'NavYaml' });
+  const allNavYamlNodes = nodeModel
+    .getAllNodes({ type: 'NavYaml' })
+    .sort((a, b) => a.path.localeCompare(b.path));
```

Comment:
We might want to compare against the title rather than the path, since _technically_ the path isn't required (nav sections where the section title doesn't have a link):
![Screen Shot 2021-07-15 at 11 49 01](https://user-images.githubusercontent.com/1946433/125841344-3c06d925-9e96-4eb8-9bb3-c13ef17e0f9a.png)

Reply:
Good call! I updated it to sort on title instead.